docs/workspaces/index.md

```diff
@@ -154,6 +152,45 @@ A minimal `question.html` for an externally graded workspace should look somethi
 </pl-submission-panel>
 ```
 
+### Creating files in the workspace home directory
+
+Workspace questions can optionally include a `workspace/` subdirectory within the regular [PrairieLearn question directory structure](../question.md#directory-structure). If this `workspace/` subdirectory exists, its contents will be copied into the home directory of the student's workspace container, as configured in the `home` setting in `info.json`.
```

Comment:
Can we specify which file is used if a file appears in both the `workspace/` directory and the `_files` parameter? I think the dynamic files will overwrite files in `workspace/`?

Reply:
Done in 2119568e0.